test/stub.go

```diff
@@ -98,6 +98,16 @@ func (s *StubBoletoRequest) WithBuyerZipCode(zipcode string) *StubBoletoRequest
 	return s
 }
 
+func (s *StubBoletoRequest) WithRecipientDocumentType(DocumentType string) *StubBoletoRequest {
+	s.Recipient.Document.Type = DocumentType
+	return s
+}
+
+func (s *StubBoletoRequest) WithRecipientName(RecipientName string) *StubBoletoRequest {
+	s.Recipient.Name = RecipientName
```

Comment:
Opa, o padrão de parâmetro da função é Camel case 😀  
```suggestion
	s.Recipient.Name = recipientName
```